packages/vanilla-components/src/components/progress-indicator/progress-indicator.tsx

```diff
@@ -0,0 +1,211 @@
+import { Component, Prop, State } from '@stencil/core';
+import ProgressBar from 'progressbar.js';
+import { API } from '../../services/WidgetHost';
+import { css } from 'emotion';
+import { getStats } from '../../services/StatPaths';
+
+interface Stats {
+  total: any,
+  progress: any
+}
+
+
+@Component({
+  tag: 'sqh-progress-indicator',
+  styleUrl: 'progress-indicator.scss'
+})
+export class ProgressIndicator {
```

Comment:
Since we can use `kebab-case` for our properties, I think that we should. Future Jorge will thank us.